tests/job-kill/00-local/suite.rc

```diff
@@ -2,29 +2,31 @@
 [cylc]
    [[reference test]]
        required run mode=live
-       expected task failures = t1.1, t2.1
-       live mode suite timeout = PT2M
+       expected task failures = t1.1, t2.1, t3.1, t4.1, stop2.1
+       live mode suite timeout = PT1M
 [scheduling]
     [[dependencies]]
         graph="""
-t1:start=>stop
-t2:start=>stop
-"""
+           t1:start & t2:start => stop1 =>
+              t3:start & t4:start => stop2
+           stop2:fail => shutdown"""
 [runtime]
     [[T]]
         script=sleep 120 & echo $! >file; wait
 {% if "CYLC_TEST_HOST" in environ and environ["CYLC_TEST_HOST"] %}
         [[[remote]]]
             host={{environ["CYLC_TEST_HOST"]}}
 {% endif %}
-    [[t1]]
+    [[t1, t2, t3, t4]]
         inherit=T
-        [[[job]]]
-            batch system=at
-    [[t2]]
-        inherit=T
-    [[stop]]
+    [[stop1]]
+        script="""
+# Kill t1.1 and t2.1 explicitly.
+cylc kill $CYLC_SUITE_NAME t1.1 t2.1 || true"""
+    [[stop2]]
         script="""
-cylc kill "$CYLC_SUITE_NAME" 't1.1' 't2.1' || true
-cylc stop "$CYLC_SUITE_NAME"
-"""
+# Kill t3.1, t4.1, and myself! implicitly (kill all active tasks).
+cylc kill $CYLC_SUITE_NAME || true
```

Comment:
Nice! No need to test at work tomorrow. Can see it was tested here in the functional tests! :tada: